packages/mdc-textfield/index.js

```diff
@@ -113,6 +113,14 @@ class MDCTextfield extends MDCComponent {
     this.foundation_.setValid(valid);
   }
 
+  get helptextValue() {
```

Comment:
Remember to annotate for Closure:  https://material.io/components/web/docs/closure-compiler/

Reply:
Good catch, thanks!